lib/platform/gitlab/index.ts

```diff
@@ -511,25 +531,43 @@ export async function ensureIssueClosing(title: string) {
   }
 }
 
-export async function addAssignees(iid: number, assignees: string[]) {
+export async function addAssignees(
+  iid: number,
+  assignees: string[]
+): Promise<any> {
   logger.debug(`Adding assignees ${assignees} to #${iid}`);
   try {
-    const assigneeId = (await api.get(`users?username=${assignees[0]}`)).body[0]
+    let assigneeId = (await api.get(`users?username=${assignees[0]}`)).body[0]
       .id;
-    let url = `projects/${config.repository}/merge_requests/${iid}`;
-    url += `?assignee_id=${assigneeId}`;
+    let url = `projects/${config.repository}/merge_requests/${iid}?assignee_id=${assigneeId}`;
     await api.put(url);
+    try {
+      if (assignees.length > 1) {
+        url = `projects/${config.repository}/merge_requests/${iid}?assignee_ids[]=${assigneeId}`;
+        for (let i = 1; i < assignees.length; i += 1) {
+          assigneeId = (await api.get(`users?username=${assignees[i]}`)).body[0]
+            .id;
+          url += `&assignee_ids[]=${assigneeId}`;
+        }
+        await api.put(url);
```

Comment:
If I understand it correctly, this second PUT will essentially override the existing assignees for the issue?

Reply:
Yes, that way even if it fails the "happy path" should be respected and at least 1 assignee should be in the PR